metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -123,6 +143,9 @@ class SupersetConfig(
         description="Can be used to change mapping for database names in superset to what you have in datahub",
     )
 
+    class Config:
+        extra = "allow"
```

Comment:
can we add a quick test to ensure that preset configs can still be parsed properly

we have some other tests that are similar e.g. test_default_convert_column_urns_to_lowercase for dbt